homework_25/src/main.rs

```diff
@@ -0,0 +1,94 @@
+use std::fmt;
+use std::ops::{Add, AddAssign};
+
+#[derive(Debug)]
+enum Pet {
+    Dog(String),
+    Cat(Cat),
+}
+
+impl From<Cat> for Pet {
+    fn from(cat: Cat) -> Pet {
+        Pet::Cat(cat)
+    }
+}
+
+#[derive(Clone, Debug)]
+struct Cat {
+    age: u32,
+    name: String,
+}
+
+impl Cat {
+    pub fn get_name(&self) -> &str {
```

Comment:
В Rust геттеры принято писать без приставки `get`)

```suggestion
    pub fn name(&self) -> &str {
```